Curvature/HTTPRequest/HTTPRequest.swift

```diff
@@ -32,6 +32,7 @@ public struct HTTPRequest {
 
     public var parameters: [String: String] = [:]
     public var data: [String: Any] = [:]
+    public var context: [String: Any] = [:]
```

Comment:
For now, you can change `data` to `context` and I'll update Camarilla to use it. (: